projects/angular/src/modal/side-panel.ts

```diff
@@ -109,30 +120,44 @@ export class ClrSidePanel implements OnInit, OnDestroy, OnChanges {
     this._pinnable = pinnable;
   }
 
+  @ViewChild(ClrModal)
+  private get modal(): ClrModal {
+    return this._modal;
+  }
+
+  private set modal(modal: ClrModal) {
+    this._modal = modal;
+    this.originalStopClose = this.modal.stopClose;
+    this.updateModalState();
+  }
+
   private get hostElement(): HTMLElement {
     return (this.element.nativeElement as HTMLElement).closest('.clr-modal-host') || document.body;
   }
 
-  ngOnInit(): void {
-    this.configuration.fadeMove = 'fadeLeft';
+  @HostBinding('class.side-panel-bottom')
+  private get bottomPositionCssClass() {
+    return this.position === 'bottom';
```

Comment:
Can we add it directly to the meta data of the component under host: {}?
```suggestion
'[class.side-panel-bottom]': 'position === 'bottom',
```

Reply:
I proposed it to keep it consistent with that we already have a class set like that here and also on other components we are generally using that for classes